src/index.html

```diff
@@ -7,5 +7,32 @@
     <title>Figma Land</title>
   </head>
 
-  <body></body>
+  <body>
+    <!-- START HEADER SECTION -->
+    <header class="header_container">
+      <a href="javascript:void(0)" class="header_logo">
+        <img src="assets/images/logo.png" alt="Logo" class="header_logo_img" />
+        <span class="header_logo_text">Product</span>
+      </a>
+      <nav class="header_navbar">
+        <ul class="header_list">
+          <li>
+            <a href="javascript:void(0)" class="header_link" style="font-weight: var(--fw-semibold)">Product</a>
```

Comment:
Why you use the styles directly in the HTML ? Should update

Reply:
Thank you, i updated it